.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!-- Before opening an issue here make sure, that you have read the template completly through -->
+
+When asking general "how to" questions:
+
+- Please do not open an issue here!
+- Instead, ask for help on [StackOverflow](stackoverflow.com/) with Materialize and Angular tags.
```

Comment:
stackoverflow link doesn't seems to work

Reply:
Fixed in new commit.